src/views/product-tutorials-view/index.tsx

```diff
@@ -12,14 +18,30 @@ import s from './product-tutorials-view.module.css'
 function ProductTutorialsView({
   layoutProps,
   data,
-  product,
 }: ProductTutorialsViewProps): React.ReactElement {
+  const currentProduct = useCurrentProduct()
```

Comment:
I do think it makes more sense to use the `product` props that are passed to this view.

Reply:
I took this into consideration before choosing the hook, and ultimately decided to go with the hook because I think it benefits us more in the long term.